### ISSUE TITLE
[invalid] Move project root

### ISSUE THREAD

**Issue by yoonthegoon** [Sansom777/CookingNinja#6]
# Move project root

## Description

Project root is currently in `CookingNinja` with contents:

```tree
.
├── .DS_Store
├── Cook
├── README.md
└── node_modules
```

However, the actual project is in `CookingNinja/Cook/cooking-ninja` with contents:

```tree
.
├── .gitignore
├── README.md
├── cooking-ninja
├── data
├── package-lock.json
├── package.json
├── public
└── src
```

It is clear that this is the intended project root as this is where `.gitignore` is.  
This is also typically the standard project root for a React project.

## Steps to resolve

First, make sure there are no changes that need to be pushed.  
In the current project root locally, execute the following:

```sh
cd Cook/cooking-ninja
git init
git add .
git commit -m "resolves #6 | Move project root"
git branch -M main
git remote add origin https://github.com/Sansom777/CookingNinja.git
git push -u origin main
```

Then, re-open the project in your IDE so that the root is now `Cook/cooking-ninja`.  
Finally, delete the old project root, first making sure that you are in the correct directory, that you have no uncommitted changes, and that you have pushed your changes to GitHub:

```sh
cd ../..
rm -rf .git
```

**Comment by yoonthegoon** [comment]
Don't forget to label this issue `invalid`.  
It's important you add `resolves #6` in the commit message, as that will link the commit to this issue, and when it gets merged, will automatically close this issue.  
Also please make sure everything looks right before you remove `.git` from your current project root.

**Comment by yoonthegoon** [comment]
Another thing to note is that I know adding a `resolves` or `fixes` or any such keywords in the commit messages will automatically close the linked issue when merged.  
However, I have you pushing to main, so maybe it won't close automatically.  
You can either create and push to a new branch to first be merged, or you may have to manually close this issue after you push, I don't know.

**Comment by yoonthegoon** [comment]
resolved by #10
